src/Sarif.Converters.UnitTests/TestData/FortifyFprConverter/ExpectedOutputs/OneResultBasic.sarif

```diff
@@ -115,31 +101,45 @@
           ]
         }
       ],
-      "files": [
+      "tool": {
+        "driver": {
+          "name": "HP Fortify Static Code Analyzer",
+          "ruleDescriptors": [
+            {
+              "id": "31D4607A-A3FF-447C-908A-CA2BBE4CE4B7",
+              "shortDescription": {
+                "text": "Sample abstract text. SQL injection vulnerability."
+              },
+              "fullDescription": {
+                "text": "The quick brown fox jumps over the lazy dog.\nThis section explains the rule in detail."
+              }
+            }
+          ]
+        }
+      },
+      "invocations": [
+        {
+          "commandLine": "[REMOVED]insourceanalyzer.exe -scan -b The_Build_ID -machine-output -f C:\\projects\\myproject\\scans\\2018-01-01_12:00:00\\Scan.fpr -format fpr",
+          "startTimeUtc": "2018-01-01T12:00:00.000Z",
+          "machine": "SCANMACHINE",
+          "account": "johndoe",
+          "properties": {
+            "Platform": "Linux"
+          }
+        }
+      ],
+      "artifacts": [
         {
-          "fileLocation": {
+          "location": {
             "uri": "Helpers/InvoiceHelper.cs",
             "uriBaseId": "SRCROOT",
-            "fileIndex": 0
+            "index": 0
```

Comment:
>"index": 0 [](start = 12, length = 10)

this data shouldn't be here but fixing the problem will also create deltas in hundreds of converted test files. so i'm deferring it.

[https://github.com/Microsoft/sarif-sdk/issues/1291](https://github.com/Microsoft/sarif-sdk/issues/1291)